src/data/data1d/filter1d/autoPhaseCorrection.js

```diff
@@ -12,17 +12,17 @@ export const name = 'Automatic phase correction';
  */
 
 const defaultOptions = {
-  magnitudeMode: false,
-  minRegSize: 30,
-  maxDistanceToJoin: 256,
+  minRegSize: 10,
```

Comment:
Could you change the variable name ? Is this minResionSize ?

Reply:
I could

Reply:
So, we will need to wait for the new release of ml-spectra-processing